src/tests/bdd/document/update_with_blob.feature

```diff
@@ -88,6 +88,7 @@ Feature: Update document that has blob data
           "blob": {
             "name": "file1",
             "type": "dmss://system/SIMOS/Blob",
+            "_blob_id": "1",
```

Comment:
Think we should change the blueprint for "Blob", or create a special rule. "_blob_id" should be created on backend I believe